tests/api_tests/nodes/test_views.py

```diff
@@ -1025,7 +1533,7 @@ def test_creates_child_logged_in_owner(self):
         assert_equal(res.json['data']['id'], self.project.nodes[0]._id)
 
     def test_creates_child_logged_in_write_contributor(self):
-        self.project.add_contributor(self.user_two, permissions=['read', 'write'], auth=Auth(self.user), save=True)
+        self.project.add_contributor(self.user_two, permissions=[permissions.READ, permissions.WRITE], auth=Auth(self.user), save=True)
```

Comment:
Ah, wait, this was a call to add_contributor, not post_json. My mistake, and ditto for the similar comment below.